src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java

```diff
@@ -0,0 +1,28 @@
+/*xml
+<module name="Checker">
+  <module name="SuppressionSingleFilter">
+    <property name="checks" value="JavadocStyleCheck"/>
```

Comment:
Please add `JavadocStyleCheck` to config as module.
Same for all other modules that we try to suppress.